src/Welcome.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import getToken from './lib/tokens';
+
+const Welcome = () => {
+  const storeToken = () => {
+    const token = getToken();
+    localStorage.setItem('userToken', token);
+    window.location.replace('/list');
```

Comment:
Using `window.location.replace` will cause the entire app to reload since it's changing the location through the browser. It would be better to use React Router for navigation to improve the user experience and prevent the page reloading from happening. You can do this with the [useHistory](https://reactrouter.com/web/api/Hooks/usehistory) hook 🪝

Reply:
Oh, nice! We knew there probably was a better way to do this, but we couldn't find it ourselves. Thank you!

Reply:
Didn't know you could change the location this way, but very good to know!

Reply:
@luisaugusto thanks for the suggestion! It looks much better now.